### PR TITLE
Feature/better buttons good form bad item

### PR DESCRIPTION
**Pull request by danielmurray**
* improved button interactions
* layed groundwork for form, input lists and list items

### CODE REVIEW

**Comment by danielmurray** [comment]
New PR on woodjs

File: src/js/appbar.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
     Wood.AppBar = Marionette.LayoutView.extend({
       tagName: "appbar",
       attributes: {
-        class: 'appbar',
+        class: 'wood appbar',
```
Comment (erichulser) [inline src/js/appbar.js]: you like this better than:

    <div class="wood">
         
         <appbar class="appbar">
    </div>

also, any reason to use `appbar` as the tag and class?

File: tests/example.js [hunk 2]
```diff
@@ -59,6 +61,32 @@
         var raisedButton = new Wood.RaisedButton();
         this.raisedButtonContainer.show(raisedButton);
 
+        // Form
+        var form = new Wood.Form({
+          inputs: [{
```
Comment (erichulser) [inline tests/example.js]: noice.

File: src/js/navbar.js [hunk 3]
```diff
@@ -1,41 +0,0 @@
-(function (driver) {
```
Comment (erichulser) [inline src/js/navbar.js]: this is now appbar yea?

File: src/js/appbar.js (hunk 1 above)
Comment (danielmurray) [inline src/js/appbar.js]: I think how I am going to solve this namespace issue is through the tagName
wood-appbar
wood-button
.. this is how polymer and react do it... namespacing through the component name

File: src/js/navbar.js (hunk 3 above)
Comment (danielmurray) [inline src/js/navbar.js]: sorry left over vestiges

**Pull request merged**